backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -31,6 +33,8 @@ public UserDto registerUser(UserDto userDto) {
       throw new UserException(UserExceptionType.EMAIL_ALREADY_IN_USE);
     }
 
```

Comment:
Should this maybe have a different name, it looks like it's supposed to return a boolean this way. Maybe something along the lines of validateUser()?

Reply:
You are right.. Will do it